lib/streamlit/elements/widgets/chat.py

```diff
@@ -46,6 +46,7 @@
 
 if TYPE_CHECKING:
     from streamlit.delta_generator import DeltaGenerator
+    from streamlit.elements.image import AtomicImage
```

Comment:
I think this can be directly imported from `image_utils` instead of `image`

Reply:
Just fixed it in the moment you commented ;)